test/ctr.bats

```diff
@@ -240,6 +242,54 @@ function teardown() {
 	stop_crio
 }
 
+@test "ctr journald logging" {
+    # ensure we have journald logging capability
+	enabled=$(check_journald)
+	if [[ "$enabled" -ne 0 ]]; then
+		skip "journald not enabled"
+	fi
+
+	start_crio_journald
+	run crictl runp "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+
+	# Create a new container.
+	newconfig=$(mktemp --tmpdir crio-config.XXXXXX.json)
+	cp "$TESTDATA"/container_config_logging.json "$newconfig"
+	sed -i 's|"%shellcommand%"|"echo here is some output \&\& echo and some from stderr >\&2"|' "$newconfig"
+	run crictl create "$pod_id" "$newconfig" "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	ctr_id="$output"
+	run crictl start "$ctr_id"
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run wait_until_exit "$ctr_id"
+	[ "$status" -eq 0 ]
+	run crictl rm "$ctr_id"
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+    # priority of 5 is LOG_NOTICE
+	journalctl -t conmon -p 5 | grep -E "here is some output$"
```

Comment:
How do we distinguish from all the pods/container running on the system?

Reply:
hm yeah lemme try to put some uniqueness into this

Reply:
fixed!